app/views/job_invocations/_form.html.erb

```diff
@@ -47,9 +47,13 @@
               <fieldset id="job_template_<%= job_template.id %>" class="job_template_form <%= 'hidden' if !@composer.job_template_ids.include?(job_template.id) && !@composer.only_one_template_available? %>">
                 <%= job_templates_fields.fields_for job_template.id.to_s do |job_template_fields| %>
                   <%= job_template_fields.fields_for :input_values do |input_values_fields| %>
-                    <% job_template.template_inputs.where(:input_type => 'user').each do |input| %>
+                    <% job_template.template_inputs.where(:input_type => ['user']).each do |input| %>
```

Comment:
just a nit, array is not needed here anymore (difference is in SQL query using `=` vs `IN`)